middleman-core/lib/middleman-core/sitemap/store.rb

```diff
@@ -156,6 +156,18 @@ def find_resource_by_destination_path(request_path)
         end
       end
 
+      # Find a resource given its destination path
```

Comment:
This doesn’t seem quite right. I would rather write: “Find index resources given its destination directory path”.